README.md

```diff
@@ -166,6 +166,91 @@ optional `--wait --timeout 15m` parameters in a deployment pipeline to see if th
 To deploy Magento to different environments (develop, staging, production) it is recommended to create a `values_*.yaml`
 for each environment and tune the resource limits and configuration values of the services.
 
+## Docker Desktop Example
+The file `values_docker.yaml` is a shorter version of `values.yaml` which contains all the changes necessary 
+to execute the chart locally on your machine and create a local cluster with the help of [Docker Desktop](https://www.docker.com/products/docker-desktop).
+The following steps will give a guide on how to achieve that:
+
+### Prerequisites
+- Install *Docker Desktop* locally (Tested with 4.10.1) and enable kubernetes in settings.
```

Comment:
@phoenix-henri "Install *Docker Desktop* >= 4.10 on your local machine and enable Kubernetes in the settings to start the cluster."